src/libuvc/libuvc.cpp

```diff
@@ -139,6 +139,10 @@ namespace librealsense
 
                         libusb_free_config_descriptor(config);
                     }
+                    else if (status < 0)
```

Comment:
I would avoid throwing from foreach_device block, since it will prevent you from discovering all devices.

Reply:
I will remove it, as it throws on MacOS, and use the previous logic (no action)